src/application.js

```diff
@@ -173,12 +199,23 @@ class XtkSchemaNode {
          */
         this.ref = EntityAccessor.getAttributeAsString(xml, "ref");
 
+        /**
+         * Has an unlimited number of children of the same type
+         * @type {boolean}
+         */
+        this.unbound = EntityAccessor.getAttributeAsBoolean(xml, "unbound");
         /**
          * Children of the node. This is a object whose key are the names of the children nodes (without the "@"
          * character for attributes) 
          * @type {Object.<string, Campaign.XtkSchemaNode>}
          */
         this.children = {};
+        /**
+         * Returns an Enumeration object which is the enumeration linked to the current node or null if there is no enumeration
+         * @type {Campaign.XtkEnumeration}
+         */
+        var nodeEnum = EntityAccessor.getAttributeAsString(xml, "enum").split(":")[2];
```

Comment:
This is unfortunately not correct. Enumerations can reference other schemas

Reply:
change reverted as discussed.